libs/remix-ui/run-tab/src/lib/actions/deploy.ts

```diff
@@ -155,14 +155,14 @@ export const createInstance = async (
     return terminalLogger(plugin, log)
   }
 
-  const finalCb = (error, contractObject, address) => {
+  const finalCb = async (error, contractObject, address) => {
     if (error) {
       const log = logBuilder(error)
 
       return terminalLogger(plugin, log)
     }
     addInstance(dispatch, { contractData: contractObject, address, name: contractObject.name })
-    const data = plugin.compilersArtefacts.getCompilerAbstract(contractObject.contract.file)
+    const data = await plugin.compilersArtefacts.getCompilerAbstract(contractObject.contract.file)
```

Comment:
Thanks for this fix! could you also check if  `getCompilerAbstract` is missing the `await` in other places in the code?

Reply:
OK, I will check it.